src/main/scala/org/constellation/API.scala

```diff
@@ -55,9 +55,13 @@ case class UpdatePassword(password: String)
 case class ProcessingConfig(
   maxWidth: Int = 10,
   minCheckpointFormationThreshold: Int = 50,
+  maxTXInBlock: Int = 50,
```

Comment:
How this whole PR is related to Wojtek's changes on the consensus flow?
By the way, afaik during the consensus flow each facilitator make a proposal of n txs which will be in the final checkpoint block. Should every facilitator pull only this `maxTXInBlock` divided by # of facilitators?
cc: @wkoszycki